framerail/src/routes/[...fallback]/+page.svelte

```diff
@@ -1 +1,12 @@
-<h1>UNTRANSLATED:General fallback route</h1>
+<!-- //[...fallback] should not be a valid page, therefore redirects to corresponding page -->
+
+<script lang="ts">
+  import { page } from "$app/stores"
+  import Redirect from "$lib/redirect.svelte"
+  let redirectPage = `/${$page.params.fallback
+    .split("/")
+    .filter((v) => v.length)
+    .join("/")}`
```

Comment:
So this will change something weird like `/scp/-/1000` into `/scp-1000`? I guess that's reasonable as a fallback route handler.

Reply:
No, it only filters out segments that are empty, for example `//page//edit` will become `/page/edit`/

Reply:
Oh you're right, I misread.